src/layouts/default.vue

```diff
@@ -163,9 +163,11 @@ export default Vue.extend({
     <Navbar class="pt-10" />
 
     <!-- Nuxt component -->
-    <Nuxt
+    <div
```

Comment:
Thanks! Let's keep it semantic HTML friendly and use a `<main>` tag instead.

Reply:
Changed it to main tag.

Reply:
Thanks.